layers/+distributions/spacemacs-base/keybindings.el

```diff
@@ -193,6 +194,16 @@
   "fvf" 'add-file-local-variable
   "fvp" 'add-file-local-variable-prop-line
   "fy" 'spacemacs/show-and-copy-buffer-filename)
+;; frame ----------------------------------------------------------------------
+(evil-leader/set-key
+  "Ff" 'find-file-other-frame
+  "Fc" 'delete-frame
+  "FC" 'delete-other-frames
+  "Fb" 'switch-to-buffer-other-frame
+  "FB" 'display-buffer-other-frame
+  "Fd" 'dired-other-frame
+  "Fo" 'other-frame
+  "Fn" 'new-frame)
```

Comment:
Is deprecated in favour of `make-frame`. Also there exists a binding on `SPC w F`